autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -247,13 +247,90 @@ def send_warnings_to_log(message, category, filename, lineno,
+          configuration (TestEvaluator)
+
+    The provided configuration determines the type of pipeline created:
+        int: A dummy classifier/dummy regressor is created. This estimator serves
+             as a baseline model to ignore all models that perform worse than this
+             dummy estimator. Also, in the worst case scenario, this is the final
+             estimator created (for instance, in case not enough memory was allocated
+             and all configurations crashed).
+        str: A traditional pipeline is created, as the configuration will contain a
+             string defining the estimator to use. For example 'RandomForest'.
+        Configuration: A pipeline object matching this configuration object is created. This
+             is the case of neural architecture search, where different backbones
+             and head can be passed in the form of a configuration object.
+
+    Attributes:
+        backend (Backend):
+            An object to interface with the disk storage. In particular, allows to
+            access the train and test datasets
+        queue (Queue):
+            Each worker available will instantiate an evaluator, and after completion,
+            it will return the evaluation result via a multiprocessing queue
+        metric (autoPyTorchMetric):
+            A scorer object that is able to evaluate how good a pipeline was fit. It
+            is a wrapper on top of the actual score method (a wrapper on top of scikit
+            lean accuracy for example) that formats the predictions accordingly.
+        budget: (float):
+            The amount of epochs/time a configuration is allowed to run.
+        budget_type  (str):
+            The budget type, which can be epochs or time
+        pipeline_config (Optional[Dict[str, Any]]):
+            Defines the content of the pipeline being evaluated. For example, it
+            contains pipeline specific settings like logging name, or whether or not
+            to use tensorboard.
+        configuration (Union[int, str, Configuration]):
+            Determines the pipeline to be constructed. A dummy estimator is created for
+            integer configurations, a traditional machine learning pipeline is created
+            for string based configuration, and NAS is performed when a configuration
+            object is passed.
+        seed (int):
+            A integer that allows for reproducibility of results
+        output_y_hat_optimization (bool):
+            Whether this worker should output the target predictions, so that they are
+            stored on disk. Fundamentally, the resampling strategy might shuffle the
+            Y_train targets, so we store the split in order to re-use them for ensemble
+            selection.
+        num_run (Optional[int]):
+            An identifier of the current configuration being fit. This number is unique per
+            configuration.
+        include (Optional[Dict[str, Any]]):
```

Comment:
Could you give a reference such as URL for pipeline steps or creating `Pipeline` object?

Reply:
There are multiple pipeline types, not a single one. For example, the traditional pipeline is different than a dummy classifier. So there is not a single pipeline we can point to?